src/systems/filecoin_blockchain/struct/tipset/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Tipset
+---
+
+{{<label tipset>}}
+Expected Consensus probabilistically elects multiple leaders in each epoch meaning a Filecoin chain may contain zero or multiple blocks at each epoch (one per elected miner). Blocks from the same epoch are assembled into tipsets. The {{<sref interpreter>}} modifies the Filecoin state tree by executing all messages in a tipset (after de-duplication of identical messages included in more than one block). 
+
+Each block references a parent tipset and validates _that tipset's state_, while proposing messages to be included for the current epoch. The state to which a new block's messages apply cannot be known until that block is incorporated into a tipset. It is thus not meaningful to execute the messages from a single block in isolation: a new state tree is only known once all messages in that block's tipset are executed. 
+ 
+A valid tipset contains a non-empty collection of blocks that all specify identical:
+
+- `Epoch` 
+- `Parents`
+- `ParentWeight`
+- `StateRoot`
+- `ReceiptsRoot`
+
+The blocks in a tipset are canonically ordered by the lexicographic ordering of the bytes in each block's ticket, breaking ties with the bytes of the CID of the block itself.
```

Comment:
may be worth being more specific here with encodings the bytes represent: littleEndian uint64 for nums, ascii for alphanumerics